daemon/src/common/Platform.h

```diff
@@ -32,7 +32,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define COMMON_PLATFORM_H_
 
 // Platform-specific configuration
-#ifdef _WIN32
+#if defined(_WIN32)
```

Comment:
I think it's fine. The rest of the code (except the include guard) uses #if defined